api/src/main/resources/liquibase-update-to-2.3.xml

```diff
@@ -33,4 +33,16 @@
     	<renameColumn tableName="allergy" oldColumnName="comment" newColumnName="comments" columnDataType="varchar(1024)"/>
     </changeSet>
 
+    <changeSet id="TRUNK-5413" author="odorajonathan" dbms="mysql">
+      <preConditions onFail="MARK_RAN">
+            <not><columnExists tableName="Concept" columnName="category"/></not>
+      </preConditions>
+      <comment>
+          Add Category field to OrderSet
+      </comment>
+      <addColumn tableName="Concept">
+            <column name="category" type="mysql"/>
```

Comment:
Hey @dkayiwa been studying the error logs here and referencing to google, I wonder what else am missing, could you guide, I want this issue done with

Reply:
Have you looked at existing changesets and seen how different they are from yours?

Reply:
Oh yes I have! A couple of them adding columns to tables are the same as mine, except for a few others that have a foreign key constraint or nullifying constraints. I think I should add a ForeignKeyConstraint then

Reply:
And what does the tableName property represent?

Reply:
As far as I know it represents name of the table in the DB

Reply:
Do you have such a table name in your DB? Or do you see any existing changeset with such a table name?

Reply:
Yes there a concept table if I search in root, i have been looking through liquibase-update-to-2.0.xml although I haven't noticed a changeset with tableName as concept yet

Reply:
Does its name match the one in your changeset?

Reply:
OK! I see the diff, lemme re-check the other files for a tableName as concept